Sources/Hummingbird/Middleware/TracingMiddleware.swift

```diff
@@ -53,10 +54,11 @@ public struct HBTracingMiddleware<Context: HBBaseRequestContext>: HBMiddleware {
             span.updateAttributes { attributes in
                 attributes["http.method"] = request.method.rawValue
                 attributes["http.target"] = request.uri.path
-                attributes["http.flavor"] = "\(request.version.major).\(request.version.minor)"
-                attributes["http.scheme"] = request.uri.scheme?.rawValue
-                attributes["http.user_agent"] = request.headers.first(name: "user-agent")
-                attributes["http.request_content_length"] = request.headers["content-length"].first.map { Int($0) } ?? nil
+                // TODO: Get HTTP version and scheme
```

Comment:
Commenting to mark this TODO

Reply:
We never setup the scheme before, it always defaulted to `nil`. The swift http types don't include a version so not sure how we are going to get that